### PR TITLE
Docs: fix links

### PR DESCRIPTION
**Pull request by robbymilo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/loki.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Once the result is returned, the log panel shows a list of log rows and a bar ch
 
 <div class="medium-6 columns">
   <video width="800" height="500" controls>
-    <source src="https://grafana.com/static/assets/videos/explore_loki.mp4" type="video/mp4">
+    <source src="/assets/videos/explore_loki.mp4" type="video/mp4">
```
Comment (aknuds1) [inline docs/sources/features/datasources/loki.md:74]: When I try testing the docs locally (`make docs`), I can't see the video.

Comment (robbymilo) [reply docs/sources/features/datasources/loki.md:74]: Docker image needs to be updated.

Comment (aknuds1) [reply docs/sources/features/datasources/loki.md:74]: Ah... thanks for clearing that up @robbymilo.

**Review by aknuds1 — APPROVED** [review]
LGTM

**Review by daniellee — CHANGES_REQUESTED** [review]
@robbymilo this change does not work for me locally. I built the docker image from master for the website and then ran `make docs-no-pull` for the Grafana docs.

The url on the docs page is http://localhost:3002/assets/videos/explore_loki.mp4 but it looks like it should be: http://localhost:3002/static/assets/videos/explore_loki.mp4

Am I missing something?

**Comment by daniellee** [comment]
@robbymilo I was too late with my review - it's already merged. Will this work?

**Pull request merged**